src/Core/Framework/Adapter/Cache/CacheClearer.php

```diff
@@ -156,6 +163,17 @@ public function clearHttpCache(): void
         }
     }
 
+    private function lock(\Closure $closure, string $key, int $timeToLive): void
+    {
+        $lock = $this->lockFactory->createLock('cache-clearer::' . $key, $timeToLive);
+
+        if ($lock->acquire(true)) {
```

Comment:
should we add a comment here, that the request is blocked until the lock could be acquired? I needed to look up, what the parameter's effect is